docs/Contributing/Releasing-Fleet.md

```diff
@@ -2,19 +2,33 @@
 
 ## Release process
 
-This section outlines the release process at Fleet.
+This section outlines the release process at Fleet after all EMs have certified that they are ready for release. 
 
-The current release cadence is once every three weeks and concentrates around Wednesdays.
+The current release cadence is once every three weeks. Patch versions are released as needed.
+#### Release candidate process
 
-### Release testing
+Major, minor, and patch releases go through the same release candidate and smoke testing process prior to release. 
 
-To make sure we build quality releases, Fleet has a freeze period for testing prior to each release. 
-Effective at the start of the freeze period, we will not merge new feature work.
+1. The release ritual or patch release DRI creates a release candidate branch. If this is a major or minor version, they prepend the branch name with `prepare-v`. If it is a patch version, they prepend the branch name with `patch-v`. For example, `prepare-v4.0.0` or `patch-v4.0.1`.
```

Comment:
Just thinking out loud but this could definitely be automated as a GH action manual workflow only runnable by `release DRI` (no one likes having to make sure we don't have a typo in something as simple as this, automation would help reduce that)

Reply:
I could see that as an option. Going with this because we already have the `patch-` branch name automation in place but open to changing later.